spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -54,6 +55,18 @@ The ICS 20 token denominations are represented by the form `{ics20Port}/{ics20Ch
 
 A sending chain may be acting as a source or sink zone. When a chain is sending tokens across a port and channel which are not equal to the last prefixed port and channel pair, it is acting as a source zone. When tokens are sent from a source zone, the destination port and channel will be prefixed onto the denomination (once the tokens are received) adding another hop to a tokens record. When a chain is sending tokens across a port and channel which are equal to the last prefixed port and channel pair, it is acting as a sink zone. When tokens are sent from a sink zone, the last prefixed port and channel pair on the denomination is removed (once the tokens are received), undoing the last hop in the tokens record. A more complete explanation is [present in the ibc-go implementation](https://github.com/cosmos/ibc-go/blob/457095517b7832c42ecf13571fee1e550fec02d0/modules/apps/transfer/keeper/relay.go#L18-L49).
 
+The memo is not parsed by the ICS20 application and may be used for external users or higher-level middleware and smart contracts. If the memo is intended to be used within the state machine, it must be in JSON dictionary format with the relevant information keyed for the intended user.
+
+The forwardingPath of the packet data is the path to the final destination of the tokens after they reach the immediate destination chain specified by the packet's channel identifiers. The forwardingPath may be specified as a list of port/channel identifier tuples prefixed by `channels:` or a list of chainIDs that will be resolved with an on-chain registry prefixed by `chains:`. The chainID specification may be unsupported, in which case the implementation should return an error acknowledgement. A special keyword: `origin` may be prefixed to the forwardingInfo list to instruct the forwarding logic to first unwind the token path to the origin chain before continuing with forwarding. If the specified forwardingPath is supported, the implementation will forward the path along one hop forward and asynchronously write the acknowledgement once the forwarded packet has returned.
+
+Example valid forwardingPaths:
+- `channels:transfer/channel-3/transfer/channel-2/transfer/channel-300`: On each receiving chain, the implementation will pop a (srcPortID, srcChannelID) from the front of the list, resolve to a channel, and then forward the tokens along. Once the forwardingPath is empty, it will send to specified receiver which must be a valid address at the final destination.
```

Comment:
If I understand it correctly, in this case `forwardingPath` should contain the source port and source channel for the chain between the origin and the destination, right? so if I want to send a packet chain A -> chain B -> chain C, then `forwardingPath` will only include the source port and source channel on chain B. Is that correct? If that's the case, maybe it would be good to mention it explicitly.